lib/peep_accessor.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'peep'
+require_relative 'helpers/db_connector'
+class PeepAccessor
+  def initialize; end
```

Comment:
You don't need to declare `initialize` if you're not extending the method — Ruby already does this for you in the background. We do this only if we want the class initialisation to create some state, spin up another object or run another kind of operation.

Reply:
Also if you delete this, your app will start behaving much more as expected.

Reply:
I'll have to have a _peep_ at this once we've got the final project finished!

Reply:
I'll have to have a _peep_ at this once we've got the final project finished!